YSI_Coding/y_malloc/y_malloc_funcs.inc

```diff
@@ -340,6 +371,54 @@ P:D(Malloc_NextSlot(slot));
 
 #endif
 
+/*-------------------------------------------------------------------------*//**
+ * <param name="array">Data unit to put information in.</param>
+ * <param name="index">Index in the unit.</param>
+ * <param name="value">Float value to insert</param>
+ * <remarks>
+ *  Displays errors in secure mode.
+ * </remarks>
+ *//*------------------------------------------------------------------------**/
+
+#if defined YSI_MALLOC_SECURE
+
+	Float:Malloc_GetFloat(Alloc:array, index) {
+		P:3("Malloc_GetFloat: array = %d, index = %d.", _:array, index);
+		return Malloc_Get(array, index);
+	}
+    
+#else 
+
+	P:D(Float:Malloc_GetFloat(Alloc:array, index));
+	#define Malloc_GetFloat(%1,%2) \
+		Float:Malloc_Get(%1,%2)
+
+#endif
+
+/*-------------------------------------------------------------------------*//**
+ * <param name="array">Data unit to put information in.</param>
+ * <param name="index">Index in the unit.</param>
+ * <param name="value">Float value to insert</param>
+ * <remarks>
+ *  Displays errors in secure mode.
+ * </remarks>
+ *//*------------------------------------------------------------------------**/
+
+#if defined YSI_MALLOC_SECURE
+
+	Malloc_SetFloat(Alloc:array, index, Float:value) {
+		P:3("Malloc_SetFloat called: %d, %d, %f", _:array, index, value);
+		return Malloc_Set(array, index, value);
```

Comment:
You need to actually override tags here.

Reply:
done